### PR TITLE
Applications

### PR DESCRIPTION
**Pull request by litvinets**
1. Updated application view according to the design
2. Updated info-child box (added new fields)
3. Added util for getting age
4. Update API calls to get applications by statuses
5. Updated application status enum
6. Updated API calls to get applications by workshop id
7. general style fix
8. removed unnecessary functionality

### CODE REVIEW

File: src/app/shared/store/user.state.ts [hunk 1]
```diff
@@ -76,25 +74,21 @@ export interface UserStateModel {
...
   favoriteWorkshops: Favorite[];
   favoriteWorkshopsCard: WorkshopCard[];
-  status: number;
 }
 @State<UserStateModel>({
   name: 'user',
   defaults: {
     isLoading: false,
-    workshops: Workshop[''],
+    workshops: [],
```
Comment (dmitryy90) [inline src/app/shared/store/user.state.ts:85]: how come?

File: src/app/shared/store/user.state.ts [hunk 2]
```diff
@@ -194,22 +174,13 @@ export class UserState {
...
-  }
-
-  @Action(GetApplicationsByStatus)
-  getApplicationsByStatus({ patchState }: StateContext<UserStateModel>, { payload }: GetApplicationsByStatus) {
-    return this.applicationService
-      .getApplicationsByStatus(payload)
-      .pipe(
-        tap((applications: Application[]) => {
-          return patchState({ applicationsByStatus: applications });
+          console.log(applications)
```
Comment (dmitryy90) [inline src/app/shared/store/user.state.ts:182]: console.log

File: src/app/shell/personal-cabinet/applications/application-card/application-card.component.ts [hunk 3]
```diff
@@ -20,20 +16,24 @@ export class ApplicationCardComponent implements OnInit {
 
   readonly applicationStatusUkr = ApplicationStatusUkr;
```
Comment (dmitryy90) [inline src/app/shell/personal-cabinet/applications/application-card/application-card.component.ts:17]: let use some neutral naming so we can easily add supporting of eng without refactoring

**Review by dmitryy90 — CHANGES_REQUESTED** [review]

File: src/app/shared/store/user.state.ts (hunk 1 above)
Comment (litvinets) [reply src/app/shared/store/user.state.ts:85]: I've decided to keep the same way as more other default values 
image

**Review by dmitryy90 — APPROVED** [review]

**Pull request merged**